nautobot/extras/management/commands/refresh_dynamic_group_member_caches.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+
+from nautobot.extras.models import DynamicGroup
+from nautobot.utilities.config import get_settings_or_config
+
+
+class Command(BaseCommand):
+    help = "Update the member caches for all DynamicGroups."
+
+    def handle(self, *args, **kwargs):
+        """Run through all Dynamic Groups and ensure their member caches are up to date."""
+
+        if get_settings_or_config("DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT") == 0:
+            self.stdout.write(
+                self.style.NOTICE("DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT is set to 0; skipping cache refresh")
+            )
```

Comment:
Same thought here about whether we should clear any existing caches in this case?

Reply:
Cache keys aren't readily "exposed" and we'd have to to unset/clear them here (their keys including their PKs). Worth the effort or YAGNI?

Reply:
Just trying to think of the case where we have a long non-zero timeout set and then change it to zero - worried about potential confusion/misleading behavior if we keep returning the cached values until the old timeout expires. Maybe not worth the effort.